build-support/aissemble-enforcer-extension/src/test/java/com/boozallen/aissemble/maven/enforcer/EnforceDockerExecutableSteps.java

```diff
@@ -0,0 +1,115 @@
+package com.boozallen.aissemble.maven.enforcer;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.enforcer.rule.api.EnforcerLogger;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
+
+import com.boozallen.aissemble.maven.enforcer.testenforcers.EnforceDockerExecutableTest;
+
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class EnforceDockerExecutableSteps {
+
+    private static final Path TEST_FILE_FOLDER = Paths.get("target", "test-classes", "test-files");
+    private static final Path TEST_DEFAULT_DOCKER_URL = Paths.get(String.valueOf(TEST_FILE_FOLDER), "default");
```

Comment:
A: this might be a little cleaner
```suggestion
    private static final Path TEST_DEFAULT_DOCKER_URL = TEST_FILE_FOLDER.resolve("default");
```